flutter.el

```diff
@@ -38,6 +38,9 @@
 (defvar flutter-sdk-path nil
   "Path to Flutter SDK.")
 
+(defvar flutter-args nil
+  "Space-delimited string of CLI flags passed to `flutter`.")
+
```

Comment:
There are various invocations of the flutter CLI tool (`flutter--devices`, `flutter--test`, etc.). With the name "flutter-args" I would expect this variable to apply to all of them. In practice it seems your intent is only to supply default args to `flutter-run`, so I would suggest `flutter-run-args`.